src/api_server/swagger/firecracker.yaml

```
@@ -450,6 +450,40 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /mmds/imds-version:
+    put:
+      summary: Sets IMDS version.
+      operationId: putImdsVersion
```

would it make sense to stay away from the "imds" term? It is an overloaded term somehow.

What about:
```suggestion
  /mmds/version:
    put:
      summary: Sets MMDS version.
      operationId: putMmdsVersion
```

(same for GET)

---

Done